gitbutler-ui/src/lib/components/AccountLink.svelte

```diff
@@ -5,22 +5,25 @@
 
 	export let user: User | undefined;
 	export let pop = false;
+	export let isNavCollapsed: boolean;
```

Comment:
Could we make this `export let isNavCollapsed = false` so that we can omit the parameter completely in `DecorativeSplitView.svelte`?

Reply:
ooh that is a good idea